spec/ferver_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe 'ferver' do
-  let(:file_list) { double('file-list') }
+  let(:file_list) { double('Ferver::FileList') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.